src/org/opendatakit/briefcase/export/ExportForms.java

```diff
@@ -164,7 +164,7 @@ public void putTransferSettings(FormStatus form, ServerConnectionInfo transferSe
   public boolean allSelectedFormsHaveConfiguration() {
     return getSelectedForms().stream()
         .map(this::getConfiguration)
-        .allMatch(ExportConfiguration::isValid);
+        .allMatch(ExportConfiguration::isValidAsCustomConf);
```

Comment:
As I've explained in my previous comment, the correct validation method would be isValid, because here we are checking resolved (with fallback to the default conf) configurations.